apps/src/templates/school-association.module.scss

```diff
@@ -0,0 +1,13 @@
+@import "color.scss";
+@import "font.scss";
+
+.outerContainer {
+    text-align: center !important;
```

Comment:
why is important needed here? can you leave a comment as to why?

Reply:
Yes I can leave a comment in there! Was just talking to Kelby about this and it might be worth a broader conversation. I copied from the pegasus design system spreadsheet [here](https://github.com/code-dot-org/code-dot-org/blob/staging/pegasus/sites.v3/code.org/public/css/design-system-pegasus.scss#L65) and we don't have a studio equivalent. I'm not sure if that's in our future!

Reply:
I don't think so -- we don't use unified classes much in dashboard, instead opting for unified components (when they're implemented/available).

If you remove the `!important` does it break? If it does, a comment is great! Thanks!

Reply:
Gotcha. Is this partly because of the magnitude of the work? Again, just thinking about an ideal approach from a design perspective (and learning from Kelby about how beneficial it could be to have a kind of cascading styling approach where a parent stylesheet could set up a component library for success). Anyway though I know that is beside the point here, and will either remove !important or add a comment.

Reply:
I might not be understanding -- what piece would be useful to have in dashboard? (this is not completely outside of the review of this PR -- just trying to understand!)

Reply:
For example, there are a few things in the pegasus design system like this (because I'm sure these are common needs all over the site): 
<img width="514" alt="Screenshot 2024-02-27 at 3 36 13 PM" src="https://github.com/code-dot-org/code-dot-org/assets/37230822/0d2c7272-f7dd-4352-99d6-cac36c2f7b41">